orttraining/orttraining/eager/ort_ops.h

```diff
@@ -3,16 +3,32 @@
 
 #pragma once
 
+#include "ort_util.h"
 #include <core/framework/ort_value.h>
 #include <core/eager/ort_kernel_invoker.h>
 
 namespace torch_ort {
 namespace eager {
 
+template <typename T>
 OrtValue reshape_copy(
   onnxruntime::ORTInvoker& invoker,
   const OrtValue& input,
-  std::vector<int64_t> shape);
+  T shape) {
```

Comment:
so here the shape type T should be either a std vector or dim vector, right? to make sure it is int64_t vector, you probably can define it as template class, like:
template <template<class> V >
OrtValue reshape_copy(...... V<int64_t> shape)